openforcefield/typing/engines/smirnoff/parameters.py

```diff
@@ -274,15 +274,15 @@ def __contains__(self, item):
         # Fall back to traditional access
         return list.__contains__(self, item)
 
-    def to_list(self, return_cosmetic_attributes=False):
+    def to_list(self, discard_cosmetic_attributes=True):
         """
         Render this ParameterList to a normal list, serializing each ParameterType-derived object in it to dict.
 
         Parameters
         ----------
 
-        return_cosmetic_attributes : bool, optional. default = False
-            Whether to return non-spec attributes of each ParameterType-derived object.
+        discard_cosmetic_attributes : bool, optional. Default = True
+            Whether to discard non-spec attributes of each ParameterType-derived object.
 
         Returns
         -------
```

Comment:
Are these deep copies of `ParameterType`s, or the actual objects? Would modifying an object modify the corresponding object in `ForceField`?

Reply:
These are serialized representations of ParameterTypes (so, not `ParameterType`s at all, but dicts).

Reply:
So modifying them has no effect on the corresponding object in `ForceField`.